dist/index.js

```diff
@@ -137,6 +137,15 @@ const CODE_DEPLOY_MAX_WAIT_MINUTES = 360;  // 6 hours
 const CODE_DEPLOY_MIN_WAIT_MINUTES = 30;
 const CODE_DEPLOY_WAIT_DEFAULT_DELAY_SEC = 15;
 
+// Attributes that are returned by DescribeTaskDefinition, but are not valid RegisterTaskDefinition inputs
+const IGNORED_TASK_DEFINITION_ATTRIBUTES = [
```

Comment:
why is this and `removeIgnoredAttributes` duplicated here and in `index.js`? Is this generated from the latter?

Reply:
offline: confirmed this is the case.